gateway/models.py

```diff
@@ -1,57 +1,25 @@
-# Copyright (c) 2023, Cisco Systems, Inc. and/or its affiliates.
+# Copyright (c) 2023-2024, Cisco Systems, Inc. and/or its affiliates.
```

Comment:
Do we need to change this in every file? We can probably do it with `reuse`.

Reply:
Not at this time.